app/controllers/stories_controller.rb

```diff
@@ -25,6 +25,8 @@ def index
       @events = current_user.events
       @stories = ORE::Aggregation.where(current_user_events_query.merge(index_query))
     end
+  rescue CanCan::AccessDenied
```

Comment:
If we add this to the edit action in the migration controller it will also prompt for a login there.